.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "files.associations": {
+        "string.h": "c",
+        "signal.h": "c"
```

Comment:
What's this for? And why only for these two system headers that are not part of librdkafka?
Not really sure this belongs in the repo.